packages/terra-heading/src/Heading.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+import './Heading.scss';
+
+const propTypes = {
+  /**
+   * Needs descriptions
+   */
+  children: PropTypes.node.isRequired,
```

Comment:
I've pulled the github markdown docs into the examples on the doc site:
https://terra-core-deployed-pr-470.herokuapp.com/static/#/site/heading
https://terra-core-deployed-pr-470.herokuapp.com/static/#/site/text
These docs will now be available on both GitHub and the doc site.